docs/get_started/tour.mdx

```diff
@@ -45,1015 +51,828 @@ This guide is written for:
 </TabItem>
 <TabItem value="java" label="Java">
 - JDK >= 11
-- [Docker Engine](https://docs.docker.com/engine/install/) or [Podman](https://podman.io/docs/installation) to launch the Restate runtime (not needed for the app implementation itself).
-- [curl](https://everything.curl.dev/get)
+- Restate CLI: [Installation instructions](/operate/cli#installation)
+- Optional: [Docker Engine](https://docs.docker.com/engine/install/) or [Podman](https://podman.io/docs/installation), if you want to run the Restate Server with Docker. And to run Jaeger.
 
 This guide is written for:
 - Java SDK version: `dev.restate:sdk-api:VAR::JAVA_SDK_VERSION`
```

Comment:
Maybe it's confusing if you refer to one of the specific `sdk-api` artifacts? how about you just mention the sdk version without the fully qualified artifact name?